docs/README.md

```diff
@@ -18,18 +18,17 @@
 </p>
 
 
-deck.gl is designed to simplify high-performance, WebGL-based visualization of large data sets. Users can quickly get impressive visual results with minimal effort by composing existing layers, or leverage deck.gl's extensible architecture to address custom needs.
+deck.gl is designed to simplify high-performance, WebGL-based visualization of large data sets, both geospatial and non-geospatial. Users can quickly get impressive visual results with minimal effort by composing existing layers, or leverage deck.gl's extensible architecture to address custom needs.
```

Comment:
We are a geospatial first framework. The non-geospatial support is not something we typically emphasize too much, though we are happy to see contributions on this side. Maybe slightly rephrasing this sentence along those lines?